.eslintrc.yml

```diff
@@ -6,11 +6,15 @@ globals:
   Promise: false
   Set: false
   Symbol: false
+  Proxy: false
 
 plugins:
   - ie11
   - local-rules
 
+parserOptions:
```

Comment:
Sinon is an ES5.1 project, as [described in CONTRIBUTING.md](https://github.com/sinonjs/sinon/blob/master/CONTRIBUTING.md#compatibility).

If we allow ES6 syntax, we drop support for IE11 and older Safari. I don't think we're quite there yet.